legend-engine-server/src/main/java/org/finos/legend/engine/server/ServerConfiguration.java

```diff
@@ -44,4 +48,8 @@
      */
     @Deprecated
     public TemporaryTestDbConfiguration temporarytestdb;
+
+    // applicable only for Test Server
+    public List<DatabaseType> testConnectionsToEnable = new ArrayList<>();
```

Comment:
Note to self. Look into how this can be split into a separate server/config

Reply:
done